codebuild/spec/buildspec_timing.yml

```diff
@@ -0,0 +1,36 @@
+---
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+version: 0.2
+env:
+  shell: bash
+  variables:
+    # This assumes you have a Rust toolchain installed
+    CARGO: "cargo +nightly"
+    OPENSSL_DIR: "/usr/local/openssl-3.0"
```

Comment:
Why is `OPENSSL_DIR` needed?

Reply:
It fails without this on the bare-bones Ubuntu24 container- it's likely that the `pkgconfig` files are considered optional and are in another package, like openssl-devel:

`Could not find directory of OpenSSL installation, and this `-sys` crate cannot proceed without this knowledge. If OpenSSL is installed and this crate had trouble finding it,  you can set the `OPENSSL_DIR` environment variable for the compilation process. See stderr section below for further information.
error: failed to run custom build command for `openssl-sys v0.9.107`
Caused by:
  process didn't exit successfully: `/codebuild/output/src1352710634/src/github.com/aws/s2n-tls/bindings/rust/extended/target/debug/build/openssl-sys-3f1b0c180f2b44bd/build-script-main` (exit status: 101)`

Reply:
Ah ok - I didn't realize we had a dev-dependency on openssl-sys.